Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -222,6 +222,22 @@
     <string name="disable_copy_filter_description">Disables the blending of adjacent rows when copying the EFB. This is known in some games as \"deflickering\" or \"smoothing\". Disabling the filter is usually safe, and may result in a sharper image.</string>
     <string name="arbitrary_mipmap_detection">Arbitrary Mipmap Detection</string>
     <string name="arbitrary_mipmap_detection_description">Enables detection of arbitrary mipmaps, which some games use for special distance-based effects.\nMay have false positives that result in blurry textures at increased internal resolution, such as in games that use very low resolution mipmaps. Disabling this can also reduce stutter in games that frequently load new textures.\n\nIf unsure, leave this checked.</string>
+    <string name="color_correction_submenu">Color Correction</string>
+    <string name="color_space">Color Space</string>
+    <string name="correct_color_space">Correct Color Space</string>
+    <string name="correct_color_space_description">Converts the colors to the color spaces that GC/Wii were meant to work with to sRGB/Rec.709.</string>
```

Comment:
There's a mistake in the wording of this string. Check the current version of DolphinQt for the updated wording.